code/game/objects/items/stacks/rods.dm

```diff
@@ -9,7 +9,7 @@
 	throwforce = 15.0
 	throw_speed = 5
 	throw_range = 20
-	matter = list(DEFAULT_WALL_MATERIAL = 1875)
+	matter = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT / 2)

```

Comment:
I feel like fractional matter values may cause problems down the line in regards to lathe storage etc, might be worth doing a ceil() on this.